webapp/src/main/webapp/WEB-INF/conf/onlyoffice/portal/portal-configuration.xml

```diff
@@ -52,7 +52,7 @@
               <boolean>${exo.onlyoffice.portalConfig.metadata.override:true}</boolean>
             </field>
             <field name="importMode">
-              <string>${exo.onlyoffice.portalConfig.metadata.importmode:insert}</string>
+              <string>${exo.onlyoffice.portalConfig.metadata.importmode:merge}</string>
```

Comment:
@ahamdi Shouldn't we keep the default behavior?

Reply:
@hbenali 

Thank you for your feedback, however please allow me to proceed merging this PR.
In case this needs to be restored I'll create a new one accordingly.

Thanks,